setup.py

```diff
@@ -14,60 +14,62 @@ def get_version(*file_paths):
     """Retrieves the version from django_cookies_samesite/__init__.py"""
     filename = os.path.join(os.path.dirname(__file__), *file_paths)
     version_file = open(filename).read()
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
-                              version_file, re.M)
+    version_match = re.search(
+        r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M
+    )
     if version_match:
         return version_match.group(1)
-    raise RuntimeError('Unable to find version string.')
+    raise RuntimeError("Unable to find version string.")
 
 
 version = get_version("django_cookies_samesite", "__init__.py")
 
 
-if sys.argv[-1] == 'publish':
+if sys.argv[-1] == "publish":
     try:
         import wheel
+
         print("Wheel version: ", wheel.__version__)
     except ImportError:
         print('Wheel library missing. Please run "pip install wheel"')
         sys.exit()
-    os.system('python setup.py sdist upload')
-    os.system('python setup.py bdist_wheel upload')
+    os.system("python setup.py sdist upload")
+    os.system("python setup.py bdist_wheel upload")
     sys.exit()
 
-if sys.argv[-1] == 'tag':
+if sys.argv[-1] == "tag":
     print("Tagging the version on git:")
     os.system("git tag -a %s -m 'version %s'" % (version, version))
     os.system("git push --tags")
     sys.exit()
 
-readme = open('README.rst').read()
-history = open('HISTORY.rst').read().replace('.. :changelog:', '')
+readme = open("README.rst").read()
+history = open("HISTORY.rst").read().replace(".. :changelog:", "")
 
 setup(
-    name='django-cookies-samesite',
+    name="django-cookies-samesite-fork",
     version=version,
-    description="""Middleware which sets SameSite flag for session and csrf cookies in legacy versions of Django.""",
-    long_description=readme + '\n\n' + history,
-    author='Jarek "jotes" Śmiejczak',
-    author_email='poke@jotes.work',
-    url='https://github.com/jotes/django-cookies-samesite',
-    packages=[
-        'django_cookies_samesite',
-    ],
+    description="Middleware which sets SameSite flag for session and csrf "
+    "cookies in legacy versions of Django. Forked from "
```

Comment:
I guess you keep some changes from your fork in your PR. Can you revert them? I'll merge the PR immediately after it. I want to add a file dedicated to all contributors.
Thanks in advance!

Reply:
All done!

Reply:
@antoinelb YOU are breathtaking! Thanks :dancing_men: